Question1.js

```diff
@@ -0,0 +1,38 @@
+// function Ctor() {
+// 	this.a = 0;
+// 	this.b = 1;
+// };
+
+// var obj1 = new Ctor();
+// var obj2 = new Ctor();
+
+// var obj3 = {};
+//     Ctor.call(obj3);
+
+// obj1.c = 2;
+
+// obj1.__proto__.d = 3;
+
+
+// console.log(obj1)
+// console.log(obj1.a)
+// console.log(obj3)
+// console.log(obj3.a)
+
```

Comment:
See below for the values of obj1.d, obj2.d, and obj3.d
obj1.d // 3
obj2.d // 3
obj3.d // undefined

Because obj1 and obj2 are made by new, they are automatically linked to Ctor.prototype.
But obj3 is made (implicitly) with 'new Object', so its proto is Object.prototype,
and it will not inherit from Ctor.prototype.